CHANGELOG.md

```diff
@@ -69,6 +69,11 @@ Dropping a requirement of a major version of a dependency is a new contract.
 [JPERF-416]: https://ecosystem.atlassian.net/browse/JPERF-416
 [JPERF-421]: https://ecosystem.atlassian.net/browse/JPERF-421
 
+### Fixed
```

Comment:
Note that after the rebase, this change log needs to be moved up to the `Unreleased` section